detect_circles.py

```diff
@@ -2,15 +2,18 @@
 import cv2 as cv
 
 
-vidcap = cv.VideoCapture('test_photos_balls/yellowcheck.mp4')
+#vidcap = cv.VideoCapture('test_photos_balls/yellowcheck.mp4')
```

Comment:
Let's get the ball detection stuff wrapped in the correct classes from the beginning.